### DISCUSSION TITLE
java and omz

### DISCUSSION THREAD

**Discussion by banditFox** [ohmyzsh/ohmyzsh#11496] [Help] [answered]
I installed OMZ and it's working great.
I have also installed Java and have to set the export and source home directory in ~/.zshrc

I'm very new to codeing and I'm wondering if I do this, if it's going to mess up my OMZ install.
See below:
<img width="793" alt="Screenshot 2023-02-14 at 14 10 11" src="https://user-images.githubusercontent.com/89707695/218850891-c0c44b09-4fbe-4e6a-a1ca-06743ded34ab.png">

**Comment by rwmitchell** [comment, marked as answer]
Should not

**Reply by banditFox** [reply]
Thanks so much!
